.eslintrc.js

```diff
@@ -62,6 +63,7 @@ module.exports = {
                 'camelcase': 'error',
                 'no-shadow': 'off',
                 '@typescript-eslint/no-shadow': ['error'],
+                "@typescript-eslint/no-unused-expressions": 'error', // moves eslint no-unused-expressions to a typescript error
```

Comment:
I had to turn off the one above it because it throwing an error for the TS files, if I had both on then we wouldnt be able to resolve the eslint error on the TS files even though the tslint error is gone.

Reply:
what we can do is add another section for js files specifically

Reply:
I'm indifferent here. Since we're committed to migrating everything to TS, the JS files will eventually be caught by this rule.

Reply:
Also there shouldnt be anything new as JS but as TS